syncr_backend/util/drop_util.py

```diff
@@ -0,0 +1,233 @@
+import os
+import shutil
+from typing import List
+from typing import Optional  # noqa
+from typing import Set
+from typing import Tuple
+
+from syncr_backend.constants import DEFAULT_DROP_METADATA_LOCATION
+from syncr_backend.constants import DEFAULT_FILE_METADATA_LOCATION
+from syncr_backend.init import drop_init
+from syncr_backend.init import node_init
+from syncr_backend.metadata import drop_metadata
+from syncr_backend.metadata.drop_metadata import DropMetadata
+from syncr_backend.metadata.drop_metadata import get_drop_location
+from syncr_backend.metadata.drop_metadata import save_drop_location
+from syncr_backend.metadata.file_metadata import FileMetadata
+from syncr_backend.network import send_requests
+from syncr_backend.util import crypto_util
+from syncr_backend.util import fileio_util
+
+
+def sync_drop(drop_id: bytes, save_dir: str):
+    """
+    Syncs a drop id
+
+    :param drop_id: id of the drop to sync
+    :param save_dir: save directory to save the drop
+    """
+    drop_peers = [("127.0.0.1", 1234)]
+    # TODO: get drop peers
+    raise NotImplementedError("Need to get Peers")
+
+    add_drop_from_id(drop_id, save_dir)
+    drop_metadata = get_drop_metadata(drop_id, drop_peers)
```

Comment:
add the save_dir here to avoid looking it up (it's easily accessible here so we should).